_kbarticles/2023-08-07-advantages-of-multiple-application-instances.md

```diff
@@ -0,0 +1,53 @@
+
+Multiple application instances allow customer traffic to continue being served if there are any issues with any individual application instance such as if an underlying platform component is being upgraded. Additionally having multiple application instances provides decreased chance of downtime. 
+
+Running multiple application instances allows your instances to be [scheduled across availability zones](https://docs.aws.amazon.com/whitepapers/latest/real-time-communication-on-aws/use-multiple-availability-zones.html). The increased number of instances also assists with load-balancing user requests to your application (Please note: you should set up your instances to share session data when using multiple application instances).
+
+Most recently, [as detailed in the EPA AirNow customer story](https://cloud.gov/docs/customer-stories/epa-airnow-gov/), the EPA’s AirNow program utilized the ability to quickly and easily scale up their number of application instances to handle increased customer traffic.
+
+## How it works
+
+Using the [cf push command](https://docs.cloudfoundry.org/devguide/deploy-apps/deploy-app.html#custom-cf-push), you can use the `-i` flag to indicate the number of application instances you would like.
+
+For example, pushing an application with 2 instances:
+
+```shell
+cf push myapp -i 2
+```
+
+Additionally, you can also define the number of application instances in your application `manifest.yml` file with the [`instances` manifest attribute](https://docs.cloudfoundry.org/devguide/deploy-apps/manifest-attributes.html#instances).
+
+An example for defining an application with 2 instances in your `manifest.yml` file:
+
+```shell
+    memory: 512mb
+    instances: 2
+    applications:
+        name: myapp
+        path: .
+```
+
+Please note the default number of instances is 1 instance.
+
+You can also use the [cf scale](http://docs.cloudfoundry.org/devguide/deploy-apps/cf-scale.html) command to increase the number of instances for a running app.
+
+`cf scale myapp -i 2`
+
+Please note, running multiple instances may sometimes cause scheduled tasks or data loads to run multiple times. This issue can be prevented by using the [cf-instance-index](http://docs.cloudfoundry.org/devguide/deploy-apps/environment-variable.html#CF-INSTANCE-INDEX) environment variable. This variable denotes the specific instance number.
+
+## Application instances and memory usage
+Each individual application instance utilizes the same amount of [org memory and disk reservation/limit](https://cloud.gov/docs/management/limits/) that is specified in the application manifest or indicated in the `cf push` command. Please note the application cannot access more than the specified amount of memory. 
```

Comment:
```suggestion
Each individual application instance utilizes the same amount of memory that is specified in the application manifest or indicated in the `cf push` command. Please note that the application cannot use more than the [defined memory quota for your org]({{ site.baseurl }}{% link _docs/management/limits.md %} .
```

Disk quota limits are at the application level only, not the org level

Also, all links in Jekyll posts should use the dynamic link pattern where Jekyll will dynamically generate the URL to the post, which reduces the chances of stale or inaccurate links